.gitignore

```diff
@@ -1,3 +1,4 @@
 __pycache__
 venv
-.vscode
\ No newline at end of file
+.vscode
+venv
```

Comment:
why venv double?

Reply:
where double venv
it is .vscode and venv